README.md

```diff
@@ -54,11 +62,14 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+[Docker recommends that the docker engine should start after the plugins.](https://docs.docker.com/engine/extend/plugin_api/)
+
+Starting 0.13 release, no more DEB/RPMs as vDVS(vSphere docker volume service) is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have docker 1.13/17.03 and above. 
+
+Install the plugin from Docker store by specifying “–grant-all-permissions” option. This option can be omitted to choose permissions interactively. You can customize the plugin name using “–alias” option.
 
 ```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
+docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:0.12
 ```
```

Comment:
Suggest keeping all plugin installation discussion and steps in one section and ensure no duplication of the same details.